ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml

```diff
@@ -86,6 +86,8 @@ tests:
       COMPUTE_NODE_REPLICAS: "9"
       COMPUTE_NODE_TYPE: m5.xlarge
       LOKI_USE_SERVICEMONITOR: "false"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
```

Comment:
The commit message mentions a different instance size? 
```This PR updates the Infra nodes instance types, and updating them to c5.4xlarge```

Reply:
Based on the Slack Conversation from [yesterday](https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1694530732701539), Raul will help "fit" the ingress benchmark configuration for `r5.xlarge` which is the default node-type of infra. nodes in ROSA/ Managed Service

> `This PR updates the Infra nodes instance types, and updating them to c5.4xlarge`

will update the commit msg.